src/main/java/com/cloudogu/wiki/WikiDispatcherServlet.java

```diff
@@ -78,6 +81,46 @@ protected void service(HttpServletRequest req, HttpServletResponse resp) throws
         }
     }
 
+    public List<Group> sortingWikisByGroups(Iterable<Wiki> wikis) {
+
+            /* getGroups gives a list with the groups including their repos.
+             * Therefore, the Wiki list (current order: 'repo 1 (group a), repo 2 (group a), ...')
+             * needs to be splitted into an order as 'group a (repo 1, ..), group b (repo x, ..), ..',
+             * that desired presentation in overviewRepos_en/de.html is possible.
+             */
+
+        String currentGroup = "";
+        List<Wiki> currentRepos = new ArrayList<Wiki>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS2293)